### PR TITLE
Check the running operator version in e2e

### PR DESCRIPTION
**Pull request by skitt**
Once the clusters have been deployed, check that we're actually
running the version of the operator we expect to test.

Signed-off-by: Stephen Kitt <skitt@redhat.com>

### CODE REVIEW

File: scripts/test/e2e.sh [hunk 1]
```diff
@@ -17,6 +17,18 @@ ${SCRIPTS_DIR}/e2e.sh "$@"
 # Reload KUBECONFIG in case deploy was triggered
 . "${SCRIPTS_DIR}/lib/kubecfg"
 
+# Check that we're testing the version we expect
+function check_operator_version() {
+    local opv
+    kubectl --context=$cluster logs -n submariner-operator -l name=submariner-operator --tail=-1 | grep "Submariner operator version"
```
Comment (tpantelis) [inline scripts/test/e2e.sh]: This line looks extraneous...

Comment (skitt) [reply scripts/test/e2e.sh]: Yes, it’s to check what happens here on e2e.

Comment (tpantelis) [reply scripts/test/e2e.sh]: Sorry not sure I follow....isn't it basically the same as the command run on the next line? Is this line just to print it out?

Comment (skitt) [reply scripts/test/e2e.sh]: That’s right, this line is just there so the log contents show up in the build log. I found it useful when debugging this on my own setup; if this fails to do the right thing on CI, I’d rather have this info too. I’ll remove the line before this is merged.

Comment (tpantelis) [reply scripts/test/e2e.sh]: Doesn't hurt to always print it - how about something like:

```bash
vs=$(kubectl --context=$cluster logs -n submariner-operator -l name=submariner-operator --tail=-1 | grep "Submariner operator version")
echo $vs
opv=$(awk '/$vs/ { print $NF }')
```

**Review by tpantelis — APPROVED** [review]

**Review by dfarrell07 — APPROVED** [review]

**Pull request merged**